liberty-maven-plugin/src/main/resources/net/wasdev/wlp/maven/plugins/MvnMessages.properties

```diff
@@ -301,3 +301,19 @@ error.project.not.compile.useraction=Make sure the project has been compiled.
 error.app.dependency.not.found=CWWKM2175E: Failed to install loose application. The application dependency library {0} cannot be found.
 error.app.dependency.not.found.explanation=The application dependency cannot be found in Maven local repository.
 error.app.dependency.not.found.useraction=Rebuild the project by running mvn install again.
+
+info.install.app.directory=CWWKM2176I: Installing application at {0}.
+info.install.app.directory.explanation=A request to install an application was made.
+info.install.app.directory.useraction=No action is required.
+
+error.install.app.dropins.directory=CWWKM2177E: The application is configured at {0} folder and cannot be changed to dropins folder.
+error.install.app.dropins.directory.explanation=The application is configured at {0} folder and cannot be changed to dropins folder.
+error.install.app.dropins.directory.useraction=Run the goal without appsDirectory parameter.
+
+warning.install.app.not.configured=CWWKM2178W: The application is not configured.
+warning.install.app.not.configured.explanation=The application is not configured.
+warning.install.app.not.configured.useraction=Add configuration in the source server.xml.
+
+info.install.app.add.configuration=CWWKM2179I: Adding application configuration for {0} in target server.xml.
```

Comment:
This change does not belong to this PR. I fixed the incorrect info. log number only.